Firmware/Marlin_main.cpp

```diff
@@ -3553,10 +3553,6 @@ void process_commands()
                enquecommand_P(PSTR("M24")); 
 		}	
 #ifdef FILAMENT_SENSOR
-		else if (code_seen("fsensor_recover_IR")) //! PRUSA fsensor_recover_IR
-		{
-			fsensor_restore_print_and_continue_IR();
```

Comment:
Prototype of non-existent function extern void fsensor_restore_print_and_continue_IR(void);
not removed from fsensor.h